src/main/java/com/example/helpfy/dtos/user/UserRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.example.helpfy.dtos.user;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+
+@Builder
+@Getter
+public class UserRequest {
+    @NotEmpty(message = "Email inválido.")
```

Comment:
Uma dúvida, como funcionam esses `@NotEmpty`? Eles veem o corpo da requisição e retornam erros caso não tenha os campos?

Reply:
Se não vier os campos o spring já trata isso, pois definimos o tipo que queremos receber (se vier null ele quebra). Ele verifica se não é uma string vázia. Se for, lança uma exceção.